client/checkout/classic/upe-split.js

```diff
@@ -44,6 +45,10 @@ import {
 	appendFingerprintInputToForm,
 } from '../utils/fingerprint';
 import PAYMENT_METHOD_IDS from 'wcpay/payment-methods/constants';
+import {
+	SHORTCODE_SHIPPING_ADDRESS_FIELDS,
+	SHORTCODE_BILLING_ADDRESS_FIELDS,
+} from '../../constants';
```

Comment:
Shouldn't it be? 

```suggestion
} from '../constants';
```

Does it compile with the current version?

Reply:
Fixed in 174e72c75c94d482af17642e99c98dea7de0db28.